src/components/Dashboard/TodoList.vue

```diff
@@ -603,15 +648,47 @@
             <PaymentPending v-else />
           </template>
 
+          <!-- is this an NR approved continuation draft item? -->
+          <template v-else-if="EnumUtilities.isStatusDraft(item) && EnumUtilities.isTypeContinuationIn(item)">
+            <div class="todo-list-detail body-2">
+              <p
+                v-if="getNameRequest"
+                class="list-item__subtitle"
+              >
+                NR APPROVED - {{ expiresText(getNameRequest) }}
+              </p>
+            </div>
+          </template>
+
+          <!-- is this an authorization approved continuation item? -->
+          <template v-else-if="EnumUtilities.isStatusApproved(item)">
+            <div class="todo-list-detail body-2">
+              <p
+                v-if="getNameRequest"
+                class="list-item__subtitle"
+              >
+                NR APPROVED - {{ expiresText(getNameRequest) }}
+              </p>
+              <p class="list-item__subtitle">
+                Authorization APPROVED - Expires in {} days
```

Comment:
authorization expiry date property is not implemented yet?

Reply:
Oh, this is obsolete. We removed this date. Can you update, please?

Reply:
Removed

Reply:
![image](https://github.com/user-attachments/assets/b0d4702c-ef34-4fde-b701-03ae2ec18749)